package.json

```diff
@@ -9,6 +9,7 @@
   "scripts": {
     "start": "ember server",
     "build": "ember build",
+    "postinstall": "bower install",
```

Comment:
Just a quick note here - at first this looks like a win/win for people maintaining the library but it actually hurts the ember-cli consumer app during install (and was removed in a previous PR if you look at the commit history). Could you pull this from the PR before we merge it?